### PR TITLE
Unregister notifier on shutdown to avoid an error when uninstalling

### LINKED ISSUE #947: Error after uninstalling Conversations
STB:

* install Conversations
* uninstall Conversations
* In the debug window is the following error:

Timestamp: 14.04.2015 01:03:29
Error: NS_ERROR_NOT_AVAILABLE: Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIXPCComponents_Utils.import]
Source File: resource://gre/modules/addons/XPIProvider.jsm -> jar:file:///home/paul/.thunderbird-trunk/550eddv3.deb/extensions/gconversation@xulforum.org.xpi!/bootstrap.js
Line: 148

### PR DESCRIPTION
**Pull request by pianoslum**
This should fix #947

### CODE REVIEW

**Comment by protz** [comment]
Super, will review this and others tonight. Thanks for the great series 
of fixes!

File: bootstrap.js [hunk 1]
```diff
@@ -154,6 +154,16 @@ function loadImports(){
...
+// This obserer is notified when a new window is created and injects our code
+let windowObserver = {
+  observe: function (aSubject, aTopic, aData) {
+    if (aTopic == "domwindowopened") {
+      loadImports();
+      aSubject.QueryInterface(Ci.nsIDOMWindow);
+      monkeyPatchWindow(aSubject.window, true);
+    }
+  }
+}
```
Comment (protz) [inline bootstrap.js]: missing semicolon here

**Comment by protz** [comment]
Super, thanks. That's one less error!

File: bootstrap.js (hunk 1 above)
Comment (pianoslum) [inline bootstrap.js]: And you merged anyway? Should I provide a patch?

Comment (protz) [inline bootstrap.js]: I think you can fix this in your next pull request, I don't think it
warrants a separate commit anyhow :)

~ jonathan

**Pull request merged**